app/controllers/organen/orgaan/mandatarissen.js

```diff
@@ -0,0 +1,37 @@
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { restartableTask, timeout } from 'ember-concurrency';
+import { SEARCH_TIMEOUT } from 'frontend-lmb/utils/constants';
+
+export default class OrganenMandatarissenController extends Controller {
+  @service router;
+
+  @tracked filter = '';
+  @tracked page = 0;
+  sort = 'is-bestuurlijke-alias-van.achternaam';
+  size = 20;
+
+  @tracked searchData;
+
+  @restartableTask
+  *search(searchData) {
+    yield timeout(SEARCH_TIMEOUT);
+    this.page = 0;
+    this.filter = searchData;
+  }
+
+  @action
+  selectPeriod(period) {
```

Comment:
for now this is enough, but i think this is supposed to be in the orgaan route too. That complicates things because you need to transition to the correct new (sub) route based on the one you are in right now. Not sure how easy -> new story